src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Compute</PackageId>
     <Description>Provides developers with libraries for the updated compute platform under Azure Resource manager to deploy virtual machine, virtual machine extensions and availability set management capabilities. Launch, restart, scale, capture and manage VMs, VM Extensions and more. Note: This client library is for Virtual Machines under Azure Resource Manager.</Description>
-    <VersionPrefix>16.2.0</VersionPrefix>
+    <VersionPrefix>17.0.0</VersionPrefix>
     <AssemblyName>Microsoft.Azure.Management.Compute</AssemblyName>
     <PackageTags>Microsoft Azure resource management;virtual machine;compute;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
```

Comment:
@hyonholee add packageReleaseNotes. Here is a reference
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L9

Reply:
Thanks, I added package release note.